unlock_indicator.c

```diff
@@ -91,6 +94,99 @@ static double scaling_factor(void) {
     return (dpi / 96.0);
 }
 
+/*
+ * Returns the cairo_font_face_t for sans-serif
+ * if this thing returns NULL, then we're somehow on a system without a
+ * sans-serif font, or a system without fontconfig (but with cairo somehow)
+ * Reference counts and caches the font-face, so we don't have memory leaks
+ * (and so that we don't have to go through all the parsing each time)
+ */
+
+static cairo_font_face_t *get_font_face() {
```

Comment:
nit: C requires functions taking no parameters to read `static cairo_font_face_t *get_font_face(void)`, otherwise they are interpreted as variadic functions